src/main/java/mate/academy/Main.java

```diff
@@ -41,4 +37,10 @@ public static void main(String[] args) {
 
     }
 
+    private static void createBook(BookDao bookDao, String title, BigDecimal price) {
```

Comment:
This method should only create a new book, set it fields and return created Book object. bookDao.create(book) is the method you are testing in Main, so it shouldn't be here.
createBook() method you created it's just a helper method to be used to create book objects which as this logic is being repeated several times.